### ISSUE TITLE
Create Mixxx package

### ISSUE THREAD

**Issue by icedream** [icedream/denon-prime4#6]
- [x] Create dependency packages
- [x] Create Mixxx package itself
- [x] Test full build
- [x] Test running Mixxx on framebuffer
- [x] Test running Mixxx on X
- [x] Test running Mixxx on Wayland

**Comment by icedream** [comment]
Taken care of the dependencies so far, testing a full build today once I have all the build errors out of the way.

**Comment by icedream** [comment]
So far all needed packages have been set up in Buildroot. Configuration's mostly right and I got Mixxx at least to the initial library folder select screen. Beyond that has not been happening yet though.

Still having difficulty getting Mixxx to render to the screen. I initially took the same approach Engine does: Rendering straight to framebuffer. I thought using linuxfb would do the trick, but rotation requires a manual patch and Mixxx complains about poor performance especially with rendering waveforms. It also renders way too large.

MIDI mappings missing obviously, and since all controls on the controller are pretty much MIDI inputs we get no controls outside of touchscreen just yet.

Up next is trying eglfs to make use of OpenGL for rendering.

Oddly, rebuilding partially leads to Qt randomly missing features that I already managed to compile in. Just now ran into this again after I got Mixxx to UI booting already before:

![Screenshot 2022-06-11 19-02-21](https://user-images.githubusercontent.com/807772/173197697-ffd06571-ad3a-4f67-8051-f057130c0134.png)

The image I posted on Twitter already was the same error, just with no fonts on the firmware which led to the funny effect of rendering the error message void of any text.

![Screenshot 2022-06-10 23-06-21](https://user-images.githubusercontent.com/807772/173197723-274bb6ce-297b-405d-871a-5c679cd1ad86.png)
